R/parse_obs_types.R

```diff
@@ -15,8 +15,8 @@ parse_obs_types <- function(obs) {
       tmp <- c(
         tmp,
         paste0(
-          "      ", str_if, " (obs_type[row]==", i, ")",
```

Comment:
this was the crucial bug, shouldn't have replaced `i+1` with `row`. Apparently that worked (by accident) for the tested model in the previous PR, but not for some other models.